etc/bento.env

```diff
@@ -473,3 +473,13 @@ BENTO_GRAFANA_SIGNOUT_REDIRECT_URL=https://${BENTOV2_AUTH_DOMAIN}/realms/${BENTO
 BENTO_PROMTAIL_IMAGE=grafana/promtail
 BENTO_PROMTAIL_IMAGE_VERSION=2.9.10
 BENTO_PROMTAIL_CONTAINER_NAME=${BENTOV2_PREFIX}-promtail
+
+# Minio
+BENTO_MINIO_IMAGE=quay.io/minio/minio
+BENTO_MINIO_IMAGE_VERSION=latest
```

Comment:
Generally, we want to avoid "latest" tags, since it can result on pulling different versions depending on the time.
This is fine for dev work, but before we merge this will need to be a specific version.